### ISSUE TITLE
An array of checkboxes where none are checked isn't shown with `fetch_form`

### ISSUE THREAD

**Issue by marick** [boydm/phoenix_integration#39]
*Minor bug; I'll try to have a look at it soonish*

Consider the following and note that the `name` of the second and third line ends in `[]`. 

```html
<input name="procedures[0][name]"          type="text"     value="">
<input name="procedures[0][species_ids][]" type="checkbox" value="1">
<input name="procedures[0][species_ids][]" type="checkbox" value="2">
```

That means a value is submitted with something like this:

```elixir
    |> follow_form(%{procedures:                           
                    %{0 => %{name: "procedure #1",
                             species_ids: [1]}
                    }})
```

That works fine. However, I would expect the result for `fetch_form` to look something like this:

```elixir
  inputs: %{
    _csrf_token: "XCgQR0wwBREzPykwC2Y3egtGIhp0HioOegs4xQORPuldT9s6l7O-CvsZ",
    procedures: %{
      "0": %{name: "", 
             species_ids: []}, ...     # <<<<<<
```

However, it looks like this:

```elixir
  inputs: %{
    _csrf_token: "XCgQR0wwBREzPykwC2Y3egtGIhp0HioOegs4xQORPuldT9s6l7O-CvsZ",
    procedures: %{
      "0": %{name: ""}, ...     # <<<<<<
```

**Comment by boydm** [comment]
You Sir have been using it! Great catch.

You planning to put in a fix?

**Comment by marick** [comment]
I'll work on it this week.

**Comment by boydm** [comment]
This should be fixed with PR #40
